api/edge_api/identities/views.py

```diff
@@ -0,0 +1,56 @@
+from flag_engine.identities.builders import (
+    build_identity_dict,
+    build_identity_model,
+)
+from rest_framework import status, viewsets
+from rest_framework.exceptions import NotFound
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from environments.identities.models import Identity
+from features.permissions import IdentityFeatureStatePermissions
+
+from .serializers import EdgeIdentityFeatureStateSerializer
+
+
+class EdgeIdentityFeatureStateViewSet(viewsets.ModelViewSet):
+    permission_classes = [IsAuthenticated, IdentityFeatureStatePermissions]
+    lookup_field = "featurestate_uuid"
+
+    serializer_class = EdgeIdentityFeatureStateSerializer
+
+    def initial(self, request, *args, **kwargs):
+        super().initial(request, *args, **kwargs)
+        self.identity = self._get_identity_from_request()
+
+    def _get_identity_from_request(self):
+        """
+        Get identity object from URL parameters in request.
+        """
+
+        identity_document = Identity.dynamo_wrapper.get_item_from_uuid(
+            self.kwargs["environment_api_key"],
+            self.kwargs["edge_identity_identity_uuid"],
```

Comment:
How does the FE get access to this UUID?

Reply:
From  `EdgeIdentityViewSet`, yes, that is still under `env/identtity/views`I wanted to move it here  in a different pr(since this one is about identity overrides)